examples/nomad-consul-ami/setup_ubuntu16.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -e
+
+SCRIPT=`basename "$0"`
+
+# NOTE: git is already preinstalled
+#echo "[INFO] [${SCRIPT}] Setup git"
+#sudo apt install -y git
```

Comment:
Remove this then

Reply:
This was kept for convenience since it is a real dependency it should be made visible.

Reply:
Fair enough. Could you update the comment to say "Note: git is required, but it should already be preinstalled on Ubuntu 16.0"